apps/explorer/README.md

```diff
@@ -19,7 +19,7 @@ $ pnpm install
 To start the explorer dev server, you can run the following command:
 
 ```
-pnpm explorer dev
+pnpm run dev
```

Comment:
Thank you very much for your suggestion, the problem is not in this command but the node version, it should be v20 or later

Reply:
Additionally, the command must be run from root like it states in the readme `All `pnpm` commands below are intended to be run in the root of the iota repo.` That way all dependencies will be built before building the explorer

Reply:
Do you want to update the readme yourself to state min node v20 and revert this change? Otherwise we can manage it too, thank you very much for the PR 🌷

Reply:
No worries, @begonaalvarezd 

"All pnpm commands below are intended to be run in the root of the iota repo." - I guess I got confused because I was taken straight into this folder from the documentation. It's under "app" subdirectory so I wouldn't think I've to get the whole project but my fault I should have read the documentation.

<img width="851" alt="Screenshot 2024-11-25 at 9 57 58 AM" src="https://github.com/user-attachments/assets/a885a942-10bc-4c9c-86a8-90b50fe02c3b">

You can just close this PR and do necessary change in another one. All good.

Reply:
Thank you 🌷